client/src/components/Workflow/Editor/Index.vue

```diff
@@ -191,6 +196,8 @@ import ToolPanel from "@/components/Panels/ToolPanel.vue";
 import FormDefault from "@/components/Workflow/Editor/Forms/FormDefault.vue";
 import FormTool from "@/components/Workflow/Editor/Forms/FormTool.vue";
 
+const NEW_TEMP_ID = "new_temp_workflow";
```

Comment:
Could we generate this? Maybe using uuid. Reason for this is that in the future we may want to display multiple instances of the workflow editor at once, which is the reason for the store scoping.

Reply:
Yes that is a good idea 👍

Reply:
Created a `uuid` composable that is being used instead now